### PR TITLE
Category feature implemented.

### PR DESCRIPTION
**Pull request by antony-kosenko**
- New app "categories" added;
- New viewset CategoriesAPIViewSet added (CRUD functionality);
- New model Category added as well as serializer to process a data serialization;

### CODE REVIEW

**Comment by antony-kosenko** [comment]
Ready for review and merging with the dev branch.

File: Backend/test_project/test_project/settings.py [hunk 1]
```diff
@@ -32,10 +32,13 @@
 
     # test_project apps
     'users',
+    "categories.apps.CategoriesConfig",
 
     # third-party apps
     'rest_framework',
     'djoser',
+    'mptt',
+    'django_mptt_admin',
```
Comment (llpvqll) [inline Backend/test_project/test_project/settings.py:41]: add this two libraries to requirements.txt (with versions)

**Review by llpvqll — APPROVED** [review]

File: Backend/test_project/test_project/settings.py (hunk 1 above)
Comment (antony-kosenko) [reply Backend/test_project/test_project/settings.py:41]: Done

File: Backend/test_project/test_project/settings.py [hunk 2]
```diff
@@ -32,10 +32,13 @@
 
     # test_project apps
     'users',
+    "categories.apps.CategoriesConfig",
```
Comment (Kozatskiy-Artem) [inline Backend/test_project/test_project/settings.py:36]: You can replace "categories.apps.CategoriesConfig" with 'categories'

**Review by lolesia — APPROVED** [review]

**Review by Kozatskiy-Artem — APPROVED** [review]

**Pull request merged**